### PR TITLE
Update cards functionality and Styling

### PR DESCRIPTION
**Pull request by redpath2312**
Can now update the cards once added, and improved the styling

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -5,33 +5,29 @@ import DraftCard from './Components/DraftCard'
...
 
 
-
 function App() {
 
   const [cards, setCards] = useState([]);
-  // const [doneCardsTotal, setDoneCardsTotal] = useState(0);
   const [maxIndexKey, setMaxIndexKey] = useState(0);
-  
+  const [selectedCardID, setSelectedCardID] = useState(0);
```
Comment (algren123) [inline src/App.jsx]: you might want to change the initial value from 0 to null. 0 potentially represents a card with id of 0, while null would represent no card and I think that would be more accurate in this case

File: src/Components/Card.jsx [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
-        />Done</label></div>     
-        <div><button onClick= {() => {props.onDelete(props.id)}}>Delete</button></div>    
-        </div>)
+
+
+const [isHovered, setHovered] = useState(false);
+const [cardText, setCardText] = useState("");
+
+function handleTextChange(event) {
+    const newText = (event.target.value);
```
Comment (algren123) [inline src/Components/Card.jsx]: no need for the round brackets here

**Review by algren123 — COMMENTED** [review]
Awesome job, its coming along!

Not for this PR, but I would recommend into looking at linting and formatting. ESLint is the industry standard for linting, and Prettier is the industry standard for formatting. It can be set to automatically format the file on every save.

ESLint is for showing styling or syntax issues that the default JS engine might not have.
Link: https://eslint.org/

Prettier is for formatting the files. It automatically does it and makes it much easier to read the code and makes it consistent across all files.
Link: https://prettier.io/

File: src/App.jsx (hunk 1 above)
Comment (redpath2312) [reply src/App.jsx]: Makes sense , didn't know you could set null in use state.

**Pull request merged**